src/components/AudioPlayer/AudioPlayerSlider.module.scss

```diff
@@ -44,18 +45,16 @@
 }
 
 .sliderContainerMinimized {
-  @include breakpoints.smallerThanTablet {
-    /* stylelint-disable-next-line csstools/use-logical */
-    left: 50%;
-    margin: 0 auto;
-    width: 100%;
-    transform: translate(-50%, 0);
-    position: fixed;
-    inset-block-end: constants.$audio-player-minimized-height;
-    span[role="slider"] {
-      @include breakpoints.smallerThanTablet {
-        display: none;
-      }
+  /* stylelint-disable-next-line csstools/use-logical */
```

Comment:
does this one needs to be `left` @osamasayed ,
the alternative is `inset-inline-start` if I am not wrong? does it not work with this?